src/dcurl.c

```diff
@@ -9,11 +9,10 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
-#if defined(ENABLE_OPENCL)
-#include "pow_cl.h"
-#endif
-#if defined(ENABLE_FPGA_ACCEL)
-#include "pow_fpga_accel.h"
+#ifdef __APPLE__
```

Comment:
Why did you change `src/dcurl.c`?

Reply:
The original `Doxygen` comment was written in the `dcurl.c`.
I also changed the order of the header files for better classification.

Since the comment has been moved to `dcurl.h`, I just left the rest of the modification in the commit.

Reply:
Shall I revert `dcurl.c` for a cleaner commit?

Reply:
Yes, please do. Only preserve functional changes.

Reply:
Done.